src/object-cache.cls.php

```diff
@@ -133,6 +140,23 @@ public function __construct($cfg = false)
 			$this->_cfg_enabled = false;
 		}
 	}
+	
+	/**
+	 * Add debug.
+	 *
+	 * @since  6.3
+	 * @access private
+	 */
+	private function debug_oc( $text, $show_error = false ){
+	    if( defined('LSCWP_LOG') ){
+	        Debug2::debug( $text );
+			return;
+	    }
+
+		if (!$show_error && $this->_cfg_debug!=1) return;
```

Comment:
If debug option set to `Admin IP Only`, shouldn't it still be allowed to log?

Reply:
changed